.circleci/config.yml

```diff
@@ -2285,7 +2285,7 @@ jobs:
           path: go-unit-release-junit-reports
   go-postgres-tests:
     docker:
-      - image: quay.io/rhacs-eng/apollo-ci:0.3.31
+      - image: quay.io/rhacs-eng/apollo-ci:stackrox-test-0.3.36-4-g645f6152d7
```

Comment:
Update to a tagged image?

Reply:
The PR is still in review. We may need to separate the scanner test and stackrox test image. I will update it when ready.